apps/react-workshop/src/DropdownMenu.test.ts

```diff
@@ -23,10 +23,6 @@ describe('DropdownMenu', () => {
         cy.get('button').first().click();
       });
 
-      if (testName === 'Submenu') {
-        cy.get('[role=menuitem][aria-expanded=false]').trigger('mouseenter');
```

Comment:
We should fix this test. The original intent was to open the submenu, but for some reason it wasn't working. You can see in the image that the submenu is not even open.

Reply:
Fixed (https://github.com/iTwin/iTwinUI/pull/2021/commits/6b77ac2a13b6938d7f61c2c1a7e0992a7a6f4366).